spec/models/observation_field_spec.rb

```diff
@@ -1,17 +1,30 @@
 require File.dirname(__FILE__) + '/../spec_helper.rb'
 
 describe ObservationField do
+  it { is_expected.to belong_to :user }
+  it { is_expected.to have_many(:observation_field_values).dependent :destroy }
+  it { is_expected.to have_many(:observations).through :observation_field_values }
+  it { is_expected.to have_many(:project_observation_fields).dependent :destroy }
+  it { is_expected.to have_many(:projects).through :project_observation_fields }
+  it { is_expected.to have_many(:comments).dependent :destroy }
+
+  it { is_expected.to validate_uniqueness_of(:name).case_insensitive }
+  # TODO: `validate_presence_of :name` contradicts `allow_blank` of length validation
```

Comment:
I noticed in `observation_field.rb` the seeming contradiction:
```
validates_presence_of :name
validates_length_of :name, :maximum => 255, :allow_blank => true
```
As a user, my instinct is that `allow_blank` here is no longer applicable?

Reply:
Yeah, an `ObservationField` needs a name, so I suspect we can get rid of `allow_blank`